docs/features.md

```diff
@@ -346,15 +346,16 @@ The **match** keys and **messageSrc** must use the format described in the [`jso
   },
   ```
 
-- **SSO_LOGIN** – is used to configure data necessary for the SSO login such as an issuer address, client’s ID, and scopes.
+- **SSO_LOGIN** – is used to configure data necessary for the SSO login such as an issuer address, client’s ID, client’s Secret and scopes. If clientSecret is omitted, a public client is assumed.
```

Comment:
```suggestion
- **SSO_LOGIN** – is used to configure data necessary for the SSO login such as an issuer address, client’s ID, client’s Secret and scopes. If the client's Secret is omitted, a public client is used.
```

Reply:
I'd go with `clientSecret`, as it's a name of the variable, not some client's K8s Secret resource.
```suggestion
- **SSO_LOGIN** – is used to configure data necessary for the SSO login such as an issuer address, client’s ID, client’s Secret and scopes. If `clientSecret` is omitted, a public client is used.
```
I've added backticks to emphasize that.